sandwine/_main.py

```diff
@@ -152,6 +152,13 @@ def parse_command_line(args: list[str], with_wine: bool):
         help="enable use of host X11 (CAREFUL!) (default: X11 disabled)",
     )
 
+    gpu_nvidia = parser.add_argument_group("gpu arguments")
```

Comment:
GPU is an acronym, please:

```suggestion
    gpu_nvidia = parser.add_argument_group("GPU arguments")
```

Reply:
Fixed.